en/advanced/parameters_and_configurations.md

```diff
@@ -86,6 +86,9 @@ Parameters definitions have two parts:
 Several approaches are described below for writing both the metadata and code.
 Where possible code should use newer [YAML metadata](#yaml-metadata) and [C++ API](#c-api) over the older C parameter/code definitions, as these are more flexible and robust.
 
+Parameter metadata is [compiled into the firmware](#publishing-parameter-metadata-to-a-gcs),
```

Comment:
Note that I added links here and comment above to "forward reference" the existence of the information about getting parameter to the GCS. Otherwise it is a bit invisible.